data/bingo/functions/tick/tick.mcfunction

```diff
@@ -18,19 +18,31 @@ execute if score $is_multiplayer bingo.state matches 2.. run scoreboard players
+# Send Resource pack check chat message
+#
+# Uses technical translation strings with unused argument indexes
+# (1, 2 are used indexes)
+# - "%1074992263$s%1$s" - 'bingo' encoded* as 1074992263; %1$s - 1st argument
+# - "%1074992263$s%<X>$s%1$s" - as above; version-check iteration as <X> (3+)
+# - both above translate to "%2$s" (2nd argument), in [en_us] lang-file
+# 
+# *NeunEinser's encoder: https://github.com/NeunEinser/translation-placeholder-namespace
+#tellraw @a[scores={bingo.resources=1}] ["\n\n\n\n\n", {"translate": "%1074992263$s%1$s", "with": [["",{"text": "The resourcepack is currently not enabled.", "color": "red"}, "\nIf there is a server resource pack in place, enable it and wait until the Mojang loading screen is done. Then click “re-check” down below.\n\nOtherwise, please download the Resourcepack ", {"text": "here", "color": "#00c3ff", "clickEvent": {"action": "open_url", "value": "https://github.com/NeunEinser/bingo/releases/download/5.0-pre1/Resourcepack.zip"}}, " and enable it.\n\n[", {"text": "Re-check", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.resources"}}, "]"], ["", {"translate": "%1074992263$s%3$s%1$s", "with": [["", {"text": "Wrong resourcepack version is loaded.", "color": "red"}, "\nYou are using a version of the resource pack for other version of Bingo.\n\nPlease redownload the correct Resourcepack for this Bingo version ", {"text": "here", "color": "#00c3ff", "clickEvent": {"action": "open_url", "value": "https://github.com/NeunEinser/bingo/releases/download/5.0-pre1/Resourcepack.zip"}}, ", enable it and click “re-check” below.\n\n[", {"text": "Re-check", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.resources"}}, "]"], ["", {"translate": "bingo.resourcepack_check.go_to_lobby.part1", "color": "green"}, "\n", {"translate": "bingo.resourcepack_check.go_to_lobby.part2", "with": [{"translate": "bingo.resourcepack_check.go_to_lobby.click", "color": "#00c3ff", "clickEvent": {"action": "run_command", "value": "/trigger bingo.resources set 91"}}]}]]}]]}]
```

Comment:
Hmm, thinking about this it would actually kind of cool to have this text translatable in future versions. Might need some more thought and not necessarily needed as part of this pr, but I think it should be possible like this:

- Add another translation string that has the meaning of "v3 or above" which all future versions will include. I am not sure about the pattern to use here, maybe just encode a `_up` into the namespace, or simply add another `%3$s` which just gets the definition of "or above".
- If v3 or higher use translatable string
- Otherwise fall back to hardcoded string

For this pr it would only be needed to include the translatable string in the lang file, as it obviously will never be used in the next release, only the one after this one.

Reply:
> or simply add another `%3$s` which just gets the definition of "or above".

Do you mean to just add second `<3>`? (`<value/string>` - value/encoded-string in arg adress `%<val>$s`):
```json
"<bingo><3><3><1>": "<2>"
```

Reply:
`%1074992263$s%3$s%3$s%1$s` would work for example, yes.

Reply:
- Started with https://github.com/NeunEinser/bingo/pull/111/commits/910241086d0fb469ddef353203e4c2aa9674a827
- More work in #112